html/changelogs/.all_changelog.yml

```diff
@@ -2257,3 +2257,17 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
       records and such. '
   - tweak: Smoke grenades can now have their smoke color set by using a multitool
       on them.
+2016-05-11:
+  Yoshax:
+  - tweak: Doubles the maximum length of records and names to 6144 & 52, resptively.
+      Also increases book nelngth to 12288. (All numbers are in charatcers.)

```

Comment:
May want to typo fix "resptively", "nelngth" and "charatcers", then re-run the changelog generator.